### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by robert-gleis**
Co-authored-by: tobiasrotter <info@tobias-rotter.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Starting the local database
 ```
 docker-compose up -d
 ```
```
Comment (tobiasrotter) [inline README.md:7]: test

File: README.md [hunk 2]
```diff
@@ -5,3 +5,4 @@ Starting the local database
...
```
Comment (tobiasrotter) [inline README.md:5]: test

File: README.md (hunk 1 above)
Comment (robert-gleis) [reply README.md:7]: test2

File: README.md (hunk 2 above)
Comment (robert-gleis) [reply README.md:5]: test2

**Pull request merged**